tests/integration/conftest.py

```diff
@@ -59,23 +58,14 @@ def start_fake_redis():
     # Yield control back to the tests
     yield server
 
-    # Teardown: Stop the server after all tests are finished:
+    # Teardown: Stop the server after all tests are finished
     server.shutdown()
     server.server_close()
     t.join(timeout=5)
 
 
-# mock Hana DB calls
 @pytest.fixture(scope="session")
-def mock_hana_db_connection_setup():
-    with patch("agents.kyma.tools.search.create_hana_connection") as mock_conn, patch(
-        "agents.kyma.tools.search.HanaDBRetriever", return_value=Mock()
-    ) as mock_retriever:
-        yield {"connection": mock_conn, "retriever": mock_retriever}
```

Comment:
Don't we want to mock the vector-db connection to cover more test-cases with unit-tests!

Reply:
Integration tests need real hana DB. That's why it is failing. 
I've already adapted the integration tests workflow job to add HANA credentials.
The workflow job changes will take effect after merge.
Hence, I ask you to ignore failing integration tests.